### ISSUE TITLE
todo-should-have-issue should not hard code opt-in and opt-out list

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#1372]
This was causing problems over in the number suite repos, and my best guess is because of caching. It would be good to make this more idiomatic anyways. Where you can turn it on and off for each repo. When I run on every single repo except scenery, kite, dot, and simula-rasa, here is the list. In general, there are obvious repos where we should turn things off, but many repos just have one or two TODOs, we could easily promote those to issues or just find the original committing issue. To be clear. I am NOT making this a chip away issue, but I will use this list myself. Note that I will, in most cases, just continue opting out in each package, which is nice and clear, and will allow more repos to opt into this in the future, easier.

 - [x] acid-base-solutions: @pixelzoom 1 errors in 1 files.
 - [ ] aqua: @zepumph 3 errors in 3 files.
 - [ ] area-builder: @jbphet 3 errors in 3 files.
 - [ ] area-model-common: @jonathanolson 2 errors in 1 files.
 - [ ] balancing-act: @jbphet 4 errors in 4 files.
 - [x] balancing-chemical-equations: @pixelzoom 1 errors in 1 files.
 - [ ] balloons-and-static-electricity: @jessegreenberg 1 errors in 1 files.
 - [ ] bending-light: @samreid 7 errors in 4 files.
 - [ ] build-a-nucleus: @Luisav1, @marlitas 14 errors in 10 files.
 - [ ] build-an-atom: @jbphet 9 errors in 6 files.
 - [ ] buoyancy: @jonathanolson 1 errors in 1 files.
 - [ ] calculus-grapher: @jonathanolson 7 errors in 4 files.
 - [ ] capacitor-lab-basics: @jonathanolson 12 errors in 7 files.
 - [ ] center-and-variability: @samreid, @chrisklus 61 errors in 19 files.
 - [ ] charges-and-fields: @jonathanolson 11 errors in 6 files.
 - [ ] chipper: @jonathanolson 34 errors in 3 files.
 - [ ] circuit-construction-kit-black-box-study: @samreid 23 errors in 9 files.
 - [ ] circuit-construction-kit-common: @samreid 10 errors in 7 files.
 - [ ] color-vision: @chrisklus 3 errors in 3 files.
 - [ ] coulombs-law: @zepumph 5 errors in 4 files.
 - [ ] counting-common: @chrisklus 10 errors in 6 files.
 - [ ] curve-fitting: @jonathanolson 1 errors in 1 files.
 - [ ] density-buoyancy-common: @jonathanolson 27 errors in 11 files.
 - [ ] density: @jonathanolson 1 errors in 1 files.
 - [ ] eating-exercise-and-energy: @samreid 9 errors in 6 files.
 - [ ] energy-forms-and-changes: @chrisklus 1 errors in 1 files.
 - [ ] energy-skate-park: @jessegreenberg 1 errors in 1 files.
 - [ ] estimation: @jbphet 2 errors in 2 files.
 - [ ] faradays-law: @jbphet 2 errors in 1 files.
 - [ ] fenster: @jbphet, @jessegreenberg 4 errors in 1 files.
 - [ ] fluid-pressure-and-flow: @zepumph 30 errors in 15 files.
 - [ ] forces-and-motion-basics: @jessegreenberg 32 errors in 11 files.
 - [ ] fraction-comparison: @samreid 5 errors in 4 files.
 - [ ] gravity-and-orbits: @samreid 1 errors in 1 files.
 - [ ] greenhouse-effect: @jbphet 25 errors in 22 files.
 - [ ] interaction-dashboard: @zepumph 10 errors in 7 files.
 - [ ] inverse-square-law-common: @zepumph 2 errors in 2 files.
 - [ ] john-travoltage: @jessegreenberg 3 errors in 2 files.
 - [ ] least-squares-regression: @jonathanolson 2 errors in 2 files.
 - [ ] mean-share-and-balance: @marlitas, @samreid 1 errors in 1 files.
 - [ ] mobius: @jonathanolson 1 errors in 1 files.
 - [x] models-of-the-hydrogen-atom: @pixelzoom 151 errors in 51 files.
 - [ ] molecule-shapes: @jonathanolson 10 errors in 7 files.
 - [ ] my-solar-system: @AgustinVallejo, @jonathanolson 10 errors in 7 files.
 - [x] nitroglycerin: @pixelzoom 2 errors in 1 files.
 - [ ] normal-modes:  4 errors in 3 files.
 - [ ] optics-lab: @veillette, @jbphet 2 errors in 2 files.
 - [ ] perennial: @mattpen, @jonathanolson 1 errors in 1 files.
 - [ ] phet-io-test-sim: @chrisklus 6 errors in 4 files.
 - [ ] phet-io-wrapper-lab-book: @zepumph 4 errors in 2 files.
 - [ ] phet-io-wrappers: @zepumph 18 errors in 7 files.
 - [ ] plinko-probability: @chrisklus 1 errors in 1 files.
 - [ ] quadrilateral: @jessegreenberg 40 errors in 18 files.
 - [ ] quake: @jbphet 1 errors in 1 files.
 - [ ] resistance-in-a-wire: @jbphet 1 errors in 1 files.
 - [ ] rosetta: @jbphet 1 errors in 1 files.
 - [ ] rutherford-scattering: @jessegreenberg 4 errors in 2 files.
 - [ ] shred: @jbphet 8 errors in 5 files.
 - [x] simula-rasa: @pixelzoom 18 errors in 6 files.
 - [ ] sound: @samreid 5 errors in 4 files.
 - [ ] tappi: @jessegreenberg 4 errors in 2 files.
 - [ ] wave-on-a-string: @jonathanolson 2 errors in 2 files.
 - [ ] weddell: @jbphet 10 errors in 2 files.
 - [ ] wilder: @jonathanolson 4 errors in 1 files.
 - [x] xray-diffraction: @heldentodd, @pixelzoom 3 errors in 3 files.

**Comment by zepumph** [comment]
In general, the two types of repos that I'm opting out of for having todos link issues are:
1. Very old and quite gross, and with quite a few TODOs
2. In active development, and it makes sense to not make this requirement since documentation would stifle productivity.

**Comment by zepumph** [comment]
@KatieWoe reported that aqua was failing lint on CT because I forgot that CT doesn't pull aqua, so I manually went on there to pull it.

**Comment by pixelzoom** [comment]
> ... . @pixelzoom would you like to review this in correlation to https://github.com/phetsims/number-suite-common/issues/30?

Looks good.  

It took me quite awhile to figure out how simula-rasa was opting out.  Instead of opting out in package.json, todo-should-have-issue.js is explicitly ignoring simula-rasa source files.  In the above commit, I improved the doc for this, so it makes sense to someone who may not be familiar with simula-rasa.

For repos where I'm responsible dev, I also inspected (and in some cases addressed) the TODOs that were modified by @zepumph's commits.

Back to @zepumph in case there's anything else to do here.
